### DISCUSSION TITLE
Nested css?

### DISCUSSION THREAD

**Discussion by ghost** [framesurge/perseus#232] [Q&A] [answered]
Hello. I'm experimenting with frontend on sycamore and I love it. It uses trunk.rs, which supports SASS. I would like to use perseus for SSR, but as far as I understand, I will have to abandon trunk and use perseus cli. Is it possible to use nested styles (SASS or [lightning CSS](https://github.com/parcel-bundler/lightningcss)) in perseus cli? IMHO, without them, writing something more than "hello world" is very uncomfortable

**Comment by arctic-hen7** [comment, marked as answer]
I know what you mean, it can get very inconvenient! You could easily hook up something like PostCSS to run in watch mode at the same time as the Perseus CLI, and then use the output stylesheet of that as the input into your Perseus app, that would probably be the simplest thing for now. A Sass plugin is in the works right now, but isn't ready yet unfortunately.
